Dashboard/app/js/lib/views/views.js

```diff
@@ -87,6 +87,46 @@ Ember.Handlebars.registerHelper('tooltip', function (i18nKey) {
 });
 
 
+function renderCaddisflyAnswer(json){
```

Comment:
good question. The fact that there are two files is that one is loaded when the user is logged in, and the other is loaded when the user is not logged in. However, the common elements should be able to be separated out and put in a single file.